intercom_flutter/example/android/app/build.gradle

```diff
@@ -41,7 +41,7 @@ android {
     defaultConfig {
         applicationId "io.maido.intercomexample"
         minSdkVersion 21
-        targetSdkVersion 33
+        targetSdkVersion 34
```

Comment:
Need to update the compileSdkVersion also

Reply:
Also `compileSdkVersion` is deprecated, `compileSdk` should be used instead